oci/service.go

```diff
@@ -1446,6 +1448,48 @@ func analyticsService(ctx context.Context, d *plugin.QueryData, region string) (
 	return sess, nil
 }
 
+// bastionService returns the service client for OCI Bastion service
+func bastionService(ctx context.Context, d *plugin.QueryData, region string) (*session, error) {
+	logger := plugin.Logger(ctx)
+
+	// have we already created and cached the service?
```

Comment:
```suggestion
	// Have we already created and cached the service?
```

Reply:
@ParthaI I believe we are just following the pattern from other services.